pkg/protoconv/time.go

```diff
@@ -90,3 +92,14 @@ func ConvertTimeString(str string) *gogoTimestamp.Timestamp {
 	}
 	return nil
 }
+
+// ProtoTime takes a proto time type and converts it to a human readable string down to seconds.
+// It always prints a UTC time.
+func ProtoTime(ts *gogoTimestamp.Timestamp) string {
```

Comment:
In case of an error the output is not a valid timestamp, so may be:
```suggestion
func ReadableTime(ts *gogoTimestamp.Timestamp) string {
```